martian-pipestance/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::{Context, Result};
+use serde::de::DeserializeOwned;
+use std::path::Path;
+
+pub mod common;
```

Comment:
I think it would make more sense to move the things in `common` into the crate root and mark them `pub(crate)`.